server/sonar-server/src/main/java/org/sonar/server/issue/IssueQueryFactory.java

```diff
@@ -271,7 +264,7 @@ private Date findCreatedAfterFromComponentUuid(DbSession dbSession, String uuid)
     return assignees;
   }
 
-  private boolean mergeDeprecatedComponentParameters(DbSession session, Boolean onComponentOnly,
+  private boolean mergeDeprecatedComponentParameters(DbSession session, @Nullable Boolean onComponentOnly,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)